pkg/scheduler/internal/cache/cache.go

```diff
@@ -707,3 +707,16 @@ func (cache *schedulerCache) expirePod(key string, ps *podState) error {
 func (cache *schedulerCache) NodeTree() *NodeTree {
 	return cache.nodeTree
 }
+
+// GetNodeInfo returns cached data for the node 'id'.
+func (cache *schedulerCache) GetNodeInfo(nodeID string) (*v1.Node, error) {
```

Comment:
nit: please rename `nodeID` to `name` or `nodeName`. ID may be confused with a node UID.